Modules/posixmodule.c

```diff
@@ -11433,6 +11433,32 @@ os_read_impl(PyObject *module, int fd, Py_ssize_t length)
     return buffer;
 }
 
+/*[clinic input]
+os.readinto -> Py_ssize_t
+    fd: int
+    buffer: Py_buffer(accept={rwbuffer})
+    /
+
+Read into a :ref:`buffer protocol <bufferobjects>` object from a file descriptor.
```

Comment:
Clinic docs will be rendered by `help()`, not by Sphinx, so I would prefer not to use this (well.. technically.. we can render them using autodoc but we don't use that extension)

Reply:
I thought this would feed into https://cpython-previews--129211.org.readthedocs.build/en/129211/library/os.html, but doesn't look like it. Need to find where to add for that...

Will update this for `help()`

Reply:
Since we are manually documenting functions in CPython, not using autodoc, you'll need to write the entry.